index.js

```diff
@@ -47,9 +47,17 @@ function translate(text, opts) {
         };
         data[token.name] = token.value;
 
-        return url + '?' + querystring.stringify(data);
+        var fullUrl = url + '?' + querystring.stringify(data);
+        if (fullUrl.length > 2083) {
```

Comment:
I suggest to move number to constant, e.g. `const MAX_URL_LENGTH = 2083`.

Reply:
I see where you're going with this. It could be modified to then check to see if the URL length is at or over `MAX_URL_LENGTH` instead.

Reply:
This has been resolved by taking into account @vitalets' suggestion.